examples/common.js

```diff
@@ -221,47 +143,10 @@
             }
         });
         this.getNamedLayers = function() {
-            return this.map.getLayers().getArray().filter(function(lyr) {
+            return this.storyMap.getStoryLayers().getArray().filter(function(lyr) {
```

Comment:
Yeah I was actually doubting if we still need this, since all StoryLayers will have a name.